### PR TITLE
added link to reactjs docs for test renderer

### PR DESCRIPTION
**Pull request by hartzis**
Add link to reactjs.com documentation for `react-test-renderer` to the `react-test-renderer/README.md`for fellow googlers that end up on the npm/yarn package page and think there is no documentation on how to use this thing 😸.

### CODE REVIEW

File: packages/react-test-renderer/README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ This package provides an experimental React renderer that can be used to render
 
 Essentially, this package makes it easy to grab a snapshot of the "DOM tree" rendered by a React DOM or React Native component without using a browser or jsdom.
 
+[https://reactjs.org/docs/test-renderer.html](https://reactjs.org/docs/test-renderer.html)
```
Comment (nhunzaker) [inline packages/react-test-renderer/README.md]: Minor thing, but what do you think about adding a section like `update` below:

```
Documentation:

[https://reactjs.org/docs/test-renderer.html](https://reactjs.org/docs/test-renderer.html)
```

Comment (hartzis) [reply packages/react-test-renderer/README.md]: cool! absolutely

**Comment by hartzis** [comment]
Updated. Good clarification recommendation.

**Comment by nhunzaker** [comment]
Awesome! Looks like the CI issue is unrelated. Thanks for contributing!

**Pull request merged**